pkg/yurtappmanager/apis/apps/v1alpha1/uniteddaemonset_types.go

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
file name should revised, uniteddaemonset --> yurtappdaemon

Reply:
done